### PR TITLE
show approve or preapprove to appropriate admin type

### PR DESCRIPTION
**Pull request by hobbitronics**
- Signator and Steward have different "approve" actions at different states

### CODE REVIEW

File: src/components/ClaimActions.svelte [hunk 1]
```diff
@@ -45,6 +52,9 @@ $: switch (status) {
 
 $: isEditable = editableStatuses.includes(status)
 $: showSubmit = ['Receipt', 'Revision'].includes(status) || (status === 'Draft' && needsFile)
+$: showLowerButtons =
```
Comment (RussellChamp) [inline src/components/ClaimActions.svelte:55]: is there a more descriptive name we can give this? like 'showActionButton'

File: src/components/ClaimActions.svelte [hunk 2]
```diff
@@ -82,7 +92,7 @@ const onDeny = () => dispatch('deny', message)
 </style>
 
 {#if isAdmin}
-  {#if ['Review1', 'Review2', 'Review3', 'Review3a', 'Review3b'].includes(status)}
+  {#if ['Review1', 'Review2', 'Review3'].includes(status)}
```
Comment (RussellChamp) [inline src/components/ClaimActions.svelte:95]: it would be nice to get these arrays out of the view templates, but that is probably another PR

**Review by RussellChamp — CHANGES_REQUESTED** [review]

File: src/components/ClaimActions.svelte (hunk 1 above)
Comment (hobbitronics) [reply src/components/ClaimActions.svelte:55]: they are all action buttons, but this is either preapprove or approve so we could just use `showApprovalButtons`

File: src/components/ClaimActions.svelte (hunk 2 above)
Comment (hobbitronics) [reply src/components/ClaimActions.svelte:95]: I couldn't find them already in a const or enum yet, but I agree

**Pull request merged**